src/Websites/Websites.Test/ScenarioTests/WebAppSlotTests.ps1

```diff
@@ -461,6 +461,7 @@ function Test-SetWebAppSlot
 		$slot.SiteConfig.RequestTracingEnabled = $true
 		$slot.SiteConfig.FtpsState = "FtpsOnly"
 		$slot.SiteConfig.MinTlsVersion = "1.0"
+		$slot.SiteConfig.HealthCheckPath = "/api/path"
```

Comment:
Update the tests to test using Webapp object  https://docs.microsoft.com/en-us/powershell/module/Az.Websites/set-azwebapp?view=azps-7.3.0

Reply:
Yes, I am using Webapp Object here.